modules/runtime/pages/tomcat-bundle.adoc

```diff
@@ -54,8 +54,6 @@ The Tomcat bundle is based on a standard Tomcat installation with the following
 * `server/webapps/bonita`: the Bonita web application.
 * `setup/`: a tool to manage Bonita Platform configuration, stored in database instead of filesystem. Also ships a tool to centralize all the required Tomcat bundle configuration.
 * `tools/request_key_utils: folder containing the script to generate license request keys (Subscription editions only).
-* `tools/BonitaSubscription-x.y.z`-`LDAP-Synchronizer` : folder containing the tool to synchronize your organization in Bonita with your LDAP (Subscription editions only).
-* `tools/cas-`_`cas.version`_`-module`: folder containing module files and description to enable CAS dependency to bonita EAR (Subscription editions only).
```

Comment:
As a consequence, we should remove them from the distrib and make them available somewhere else:
* ldap-synch: already done
* tools/SSO/cas: to do
* tools/SSO/kerberos: to do

Reply:
I reverted tools/CAS removal for the moment since it is still there